packages/governance-app/src/Council/CouncilCandidates.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AccountId, Vector } from '@polkadot/types';
+import { AppContext } from '@substrate/ui-common';
+import { FadedText, Header, Stacked } from '@substrate/ui-components';
+import React, { useContext, useEffect, useState } from 'react';
+import { Observable } from 'rxjs';
+
+export function CouncilCandidates () {
+  const { api } = useContext(AppContext);
+  const [councilCandidates, setCouncilCandidates] = useState();
+
+  useEffect(() => {
+    const subscription =
+      (api.query.council.candidates() as unknown as Observable<Vector<AccountId>>)
```

Comment:
Here: `(api.query.council.candidates() as Observable<Vector<AccountId>>)` should work